dominion/settings.py

```diff
@@ -108,6 +109,13 @@
     },
 }
 
+HURUMAP['dominion_countries'] = [
+  { 'geoid': 'country-KE', 'name': 'Kenya' },
+  { 'geoid': 'country-ZA', 'name': 'South Africa' },
+  { 'geoid': 'country-TZ', 'name': 'Tanzania' },
+  { 'geoid': 'country-NG', 'name': 'Nigeria' }
+];
+
```

Comment:
I used @kilemensi  list of countries in country page.
I suggest we use that list, its merged in master.